iphone/Maps/LocalizedStrings/pt-BR.lproj/Localizable.strings

```diff
@@ -1302,7 +1302,7 @@
 "enable_icloud_synchronization_title" = "Ativar a sincronização do iCloud";
 
 /* Message for the "Enable iCloud Syncronization" alert. */
-"enable_icloud_synchronization_message" = "A sincronização do iCloud é um recurso experimental em desenvolvimento. Certifique-se de que você tenha feito um backup de todos os seus favoritos e faixas.";
+"enable_icloud_synchronization_message" = "A sincronização do iCloud é um recurso experimental em desenvolvimento. Certifique-se de que você tenha feito um backup de todos os seus favoritos e trilhas.";
```

Comment:
Thanks! Please also fix translations in https://github.com/organicmaps/organicmaps/blob/master/data/strings/strings.txt

Reply:
Yep, I made a mess, so I had to do it now manually. Can you please rebase and combine multiple commits into one, please?

Reply:
Done!